non-critical-infra/hosts/hetzner-01/default.nix

```diff
@@ -6,6 +6,7 @@
       ./hardware.nix
       inputs.srvos.nixosModules.server
       inputs.srvos.nixosModules.hardware-hetzner-online-amd
+      ../modules/first-time-contribution-tagger.nix
```

Comment:
You do not need that because you already passed your module in the `flake.nix` file. You can have a modest check that things are right by running `colmena build`.

Reply:
I can run Coleman build but I still need this because it passes the settings to the module and enables it. Should I rename it?

Reply:
Oops sorry foggy brain. There is an error but the fix is actually that the path to the module is wrong.